doc/README.md

```diff
@@ -856,6 +856,29 @@ they have.
 Returns True if the event name and instance match that of the module
 checking.
 
+___format_units(value, unit='B', optimal=5, auto=True, si=False)__
```

Comment:
I've added a note.  It is about forcing the units to use rather than us trying to make them nice.

format_units(1024)  -> (1.0, 'KiB')
format_units(1024, auto=False) -> (1024, 'B')